Dockerfile

```diff
@@ -206,6 +206,7 @@ WORKDIR /node-deps
 RUN npm --no-cache install --ignore-scripts --omit=dev \
                 @salesforce/cli \
                 typescript \
+                @stoplight/spectral-cli \
```

Comment:
@nvuillam idea for improvement here: sort the required packages, to not have the order changed uselessly like here.

Reply:
100% agreed, can be done in another PR :)